apps/landingPage/src/components/footer.js

```diff
@@ -0,0 +1,97 @@
+// @flow
+
+import React from 'react';
+import { Text, Separator } from '@kiwicom/orbit-components/lib/';
+import {
+  Instagram,
+  Linkedin,
+  Facebook,
+  Twitter,
+} from '@kiwicom/orbit-components/lib/icons/';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import styled from 'styled-components';
+
+// $FlowFixMe - Cannot resolve module RelativeImageStub ( possible solution https://bit.ly/2BYfHRr )
+import logo from '../images/logo.png';
+
+const links = [
```

Comment:
These could be extract to a config file in the future, because I believe we will re-use these links throughout the page.
It's just a minor improvement, and could be done later.